migrations/074-delegate-version-row.sql

```diff
@@ -0,0 +1,3 @@
+-- Add optional version column as a number to dschief.vote_delegate_created_event table
+ALTER TABLE dschief.vote_delegate_created_event
+ADD COLUMN version INTEGER NULL;
```

Comment:
Do we also need to update the delegate graphql queries to return the new version field?